_inc/client/components/jetpack-notices/state-notices.jsx

```diff
@@ -232,14 +232,13 @@ class JetpackStateNotices extends React.Component {
 		const currentVersion = this.props.currentVersion;
 		const versionForUpgradeNotice = /(6\.8).*/;
 		const match = currentVersion.match( versionForUpgradeNotice );
-		if ( 'modules_activated' === message && match ) {
+		if ( 'modules_activated' === message && match && true === isGutenbergAvailable ) {
```

Comment:
This is trying to get the function isGutenbergAvailable() - you want `this.props.isGutenbergAvailable `

Reply:
Personal preference: I typically name my state selectors in the component something different than the selector function to avoid this confusion like 

`gutenbergAvailable: isGutenbergAvailable( state ),`